third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -198,9 +198,6 @@ public void logInformation(final String message, final Throwable exception) {
     });
   }
 
-  public void scheduleConnect() {
```

Comment:
What does the parent implementation do? Does this method (even an empty implementation) effect behavior?